client/hotrod-client/pom.xml

```diff
@@ -308,6 +302,29 @@
          <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
+            <executions>
```

Comment:
why is this required for this module and not the other hot rod client module?

Reply:
The other module does not have the same tests :)